sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_sentiment.py

```diff
@@ -0,0 +1,545 @@
+
+        docs = []
+        try:
+            result = text_analytics.analyze_sentiment(docs)
+        except HttpResponseError as err:
+            self.assertEqual(err.error_code, "MissingInputRecords")
+            self.assertIsNotNone(err.message)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    def test_text_analytics_error_duplicate_ids(self, resource_group, location, text_analytics_account, text_analytics_account_key):
+        text_analytics = TextAnalyticsClient(text_analytics_account, TextAnalyticsApiKeyCredential(text_analytics_account_key))
+        text = ""
+        for _ in range(5121):
+            text += "x"
+
+        docs = [{"id": "1", "text": ""},
+                {"id": "2", "language": "english", "text": "I did not like the hotel we stayed it."},
+                {"id": "3", "text": text}]
+
+        # Duplicate Ids
+        docs = [{"id": "1", "text": "hello world"},
+                {"id": "1", "text": "I did not like the hotel we stayed it."}]
+        try:
+            result = text_analytics.analyze_sentiment(docs)
+        except HttpResponseError as err:
+            self.assertEqual(err.error_code, "InvalidDocument")
+            self.assertIsNotNone(err.message)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    def test_text_analytics_error_batch_size_over_limit(self, resource_group, location, text_analytics_account, text_analytics_account_key):
+        text_analytics = TextAnalyticsClient(text_analytics_account, TextAnalyticsApiKeyCredential(text_analytics_account_key))
+        text = ""
+        for _ in range(5121):
+            text += "x"
+
+        docs = [{"id": "1", "text": ""},
+                {"id": "2", "language": "english", "text": "I did not like the hotel we stayed it."},
+                {"id": "3", "text": text}]
+
+        # Batch size over limit
+        docs = [u"hello world"] * 1001
+        try:
+            response = text_analytics.analyze_sentiment(docs)
+        except HttpResponseError as err:
+            self.assertEqual(err.error_code, "InvalidDocumentBatch")
+            self.assertIsNotNone(err.message)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @pytest.mark.skip(reason="Service bug returns invalidDocument here. Unskip after v3.0-preview.2")
+    def test_text_analytics_error_bad_model_version(self, resource_group, location, text_analytics_account, text_analytics_account_key):
+        text_analytics = TextAnalyticsClient(text_analytics_account, TextAnalyticsApiKeyCredential(text_analytics_account_key))
+
+        docs = [{"id": "1", "country_hint": "United States", "text": "hello world"}]
+
+        response = text_analytics.analyze_sentiment(docs)
+        self.assertEqual(response[0].error.code, "invalidCountryHint")
+        self.assertIsNotNone(response[0].error.message)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    def test_language_kwarg_spanish(self, resource_group, location, text_analytics_account, text_analytics_account_key):
```

Comment:
This test and the test below it have the same name. We just need one, this test just aims to make sure that the kwargs passed get used in the request